docs/documentation.md

```diff
@@ -69,9 +69,14 @@ To accept a different token type for the same NFT sellers must specify an altern
 
     Usually, ghost listings will cause a purchaser’s transaction to fail, which is annoying but isn’t a significant problem. We recommend using the [`cleanupPurchasedListings`](#fun-cleanupPurchasedListings) function to mitigate the issues above. 
 
-    Ghost listings could be problematic for the seller if not cleaned up specifically when the listed NFT returns to the seller’s account after the original sale or transfer out. As a result the ghost listings would once again become able to facilitate purchases. This may be undesirable as the ghost listing price may be less than fair market value at the subsequent time.
+    Ghost listings could be problematic for the seller if not cleaned up specifically when the listed NFT returns to the seller’s account after the original sale or transfer out. As a result the ghost listings would once again become able to facilitate purchases. This may be undesirable as the ghost listing price may be less than fair market value at the subsequent time. 
+
+    In order to safeguard sellers from these adverse circumstances, marketplaces have the ability to meticulously sift through a seller's entire inventory of ghost listings by utilizing the [`read_all_unique_ghost_listings`](../scripts/read_all_unique_ghost_listings.cdc) script. By executing this script at routine intervals, marketplaces can effectively purge all ghost listings with the aid of the [`cleanup_ghost_listing`](../transactions/cleanup_ghost_listing.cdc) transaction. While it is possible for any individual to cleanse a seller's ghost listings, marketplaces have the discretion to determine the degree of ease with which this process is made available to their users.
+
+    Marketplaces can assume the responsibility of eradicating all ghost listings on behalf of vendors, or alternatively, they can notify sellers to undertake the task themselves. To achieve this, they may devise an off-chain notification system to apprise users, such as sellers, of listings that warrant removal due to the nonexistence of the corresponding NFT in the seller's account. Furthermore, marketplaces may also permit any user, via their user interface, to eliminate the ghost listings of other users.
```

Comment:
This section is fairly ambiguous and unclear. It doesn't provide direction on what is recommended or best-practice, it just says 'here are options' and there's not way to understand whether any is better or worse and why you would opt to follow one or the other version

Reply:
It depends on the marketplaces which option seems fit to them.

- Many marketplaces don't want to spend the gas (Although on FLOW network it is quite cheap) in cleaning the seller's mess. They have to develop the technology piece as well to support that.
- Many marketplaces may want to give more power to the end users and want to incentives them by providing the FLOAT / special NFT that may provide the discount after 10th purchase or any way they wanna do it.
- Many marketplaces may want that seller would take care there own mess.

There is no bad choice here it depends on what kind of system a marketplace wants to build.

Reply:
Thanks for clarifying. Given the options and their persistent ambiguity we should avoid leaving things open and 'decide for yourself 🤷🏼‍♂️' since what will happen is then people do nothing, a very few do something and in general things fall apart - we'll end up with thousands of ghost listings that aren't cleaned up because we weren't firm enough in making the point that they need to be cleaned up by everyone (despite having provided all the tools to keep things in good shape). Best intentions never work - plus if we are going to bother to make the tools to clean things up then not advocating that it's done systematically seems like a major self-own. 

We should drive best-practice guidance that results in the best possible outcome for the community of users at large. To me the best-practice here is to advocate for the cleanup when a marketplace makes a sale; this seems fair since they are spending a very negligible amount of gas on something that benefits the seller they are brokers the sale for and at the time when they are making some money. Let's revise the guidance to narrow it down to explaining that marketplaces should trigger the cleanup on a sale in order to support sellers that they are transacting for. 

That said, I don't understanding how ghost listing cleanup relates to this comment "Many marketplaces may want to give more power to the end users and want to incentives them by providing the FLOAT / special NFT that may provide the discount after 10th purchase or any way they wanna do it."

Reply:
Cool I will make it a hard recommendation that marketplaces would delete the ghost listings on behalf of users. 

>That said, I don't understanding how ghost listing cleanup relates to this comment "Many marketplaces may want to give more power to the end users and want to incentives them by providing the FLOAT / special NFT that may provide the discount after 10th purchase or any way they wanna do it."

I am trying to tell that Marketplaces can create a dashboard or just a button to create ghost listings on their platform and whoever presses the button to delete the ghost listing would get incentivise (There can be many ways of incentivisation) I recommend to mint the FLOAT to the users and maybe once user has X FLOATS then the user would get discount on their platform to buy any asset from there platform. It is a gamification idea nothing else